iceoryx2/src/port/publisher.rs

```diff
@@ -53,6 +54,52 @@
 //! # Ok(())
 //! # }
 //! ```
+//!
+//! ## Sliced API
+//!
+//! ```
+//! use iceoryx2::prelude::*;
+//!
+//! # fn main() -> Result<(), Box<dyn std::error::Error>> {
+//! let service_name = ServiceName::new("My/Funk/ServiceName")?;
+//! let service = zero_copy::Service::new(&service_name)
+//!     .publish_subscribe::<[usize]>()
+//!     .open_or_create()?;
+//!
+//! let publisher = service
+//!     .publisher()
+//!     // defines the maximum length of a slice
+//!     .max_slice_len(128)
+//!     // defines how many samples can be loaned in parallel
+//!     .max_loaned_samples(5)
+//!     // defines behavior when subscriber queue is full in an non-overflowing service
+//!     .unable_to_deliver_strategy(UnableToDeliverStrategy::DiscardSample)
```

Comment:
Yes, but no. The reason is, that the overflow property is a property of the subscriber that will be set (in the future) in the builder parameters of the port.
The `UnableToDeliveryStrategy` is a property of the publisher.

This approach would make sense for me if this would be the property either of the service or only the property of the publisher port.

Reply:
Right. So this corresponds to the subscriber having the `enable_safe_overflow` option set to false.